services/proxy/pkg/command/server.go

```diff
@@ -137,7 +188,71 @@ func Server(cfg *config.Config) *cli.Command {
 	}
 }
 
-func loadMiddlewares(ctx context.Context, logger log.Logger, cfg *config.Config) alice.Chain {
+func (h *StaticRouteHandler) handler() http.Handler {
```

Comment:
Please move definition of `StaticRouteHandler` here so it and its methods are in one place.